go/base/context.go

```diff
@@ -853,3 +859,25 @@ func (this *MigrationContext) ReadConfigFile() error {
 
 	return nil
 }
+
+// getGhostTriggerName generates the name of a ghost trigger, based on original trigger name
+// or a given trigger name
+func (this *MigrationContext) GetGhostTriggerName(triggerName string) string {
+	if this.RemoveTriggerSuffix {
+		return strings.TrimSuffix(triggerName, this.TriggerSuffix)
+	}
+
+	return triggerName + this.TriggerSuffix
+}
+
+// validateGhostTriggerLength check if the ghost trigger name length is not more than 64 characters
+func (this *MigrationContext) ValidateGhostTriggerLengthBelowMaxLength(triggerName string) bool {
+	var ghostTriggerName string
+	if this.RemoveTriggerSuffix {
+		ghostTriggerName = strings.TrimSuffix(triggerName, this.TriggerSuffix)
+	} else {
+		ghostTriggerName = triggerName + this.TriggerSuffix
+	}
```

Comment:
I'm just thinking we can reuse `GetGhostTriggerName()` here, right?

Reply:
nice catch!